src/router/resolvers/Resolver.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { Route } from "vue-router";
+
+// route resolvers
+import {
+    InvokeResolver
+  } from "./index";
+
+Component.registerHooks(["beforeRouteEnter"]);
+@Component({
+    template:'<div></div>'
+})
+export default class Resolver extends Vue {
+
+  
+

```

Comment:
pls run Format Document on this guy to remove unnecessary blank lines and fix indents etc.